jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java

```diff
@@ -597,6 +600,11 @@ public static Object createEscapeHandler(JAXBDataBinding binding, String escapeH
         }
         return null;
     }
+    
+    private void addCDataWriterInterceptor(WorkItem workItem, Client client) {
+        CdataWriterInterceptor cdataWriterInterceptor = new CdataWriterInterceptor((String) workItem.getParameter("CDataElements"));
```

Comment:
I think you can save this variable and pass the object directly to the method

Reply:
I will check if CDataElements is not null to avoid registration of the interceptor in that case